Sources/Profile-View.php

```diff
@@ -446,13 +449,66 @@ function showAlerts($memID)
 
 	require_once($sourcedir . '/Profile-Modify.php');
 
+	// Are we opening a specific alert ?
+	if (!empty($_REQUEST['alert']))
+	{
+		$alert_id = (int)$_REQUEST['alert'];
+		$request = $smcFunc['db_query']('', '
+			SELECT id_member, id_member_started, content_type, content_action, content_id, is_read, extra
+			FROM {db_prefix}user_alerts
+			WHERE id_alert = {int:alert}',
+			array(
+				'alert' => $alert_id
+			)
+		);
+		$alert = $smcFunc['db_fetch_assoc']($request);
+		$smcFunc['db_free_result']($request);
+
+		// Can the user see this alert ?
+		if($context['user']['id'] != $alert['id_member'])
```

Comment:
I'm not 100% sure if this var and array is everytime setted and
on the other hand i was expecting that you use $memID.

Reply:
Huh, wonder what I was thinking there